### PR TITLE
cli: remove traces of openshift admin command

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1542332

### CODE REVIEW

File: pkg/cmd/server/admin/create_bootstrappolicy_file.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 const (
 	DefaultPolicyFile                    = "openshift.local.config/master/policy.json"
 	CreateBootstrapPolicyFileCommand     = "create-bootstrap-policy-file"
-	CreateBootstrapPolicyFileFullCommand = "openshift admin " + CreateBootstrapPolicyFileCommand
+	CreateBootstrapPolicyFileFullCommand = "oc admin " + CreateBootstrapPolicyFileCommand
```
Comment (kargakis) [inline pkg/cmd/server/admin/create_bootstrappolicy_file.go]: Is `oc admin` a thing?

Comment (mfojtik) [reply pkg/cmd/server/admin/create_bootstrappolicy_file.go]: oc adm... typo

**Comment by mfojtik** [comment]
@kargakis updated.

**Pull request merged**